### PR TITLE
Update checkpoint stress test condition

### PR DESCRIPTION
**Pull request by MishaDemianenko**
@davidegrohmann can you please take a look
Update check in checkpointing stress test to be based on percentage of tasks in desired time interval

### CODE REVIEW

File: stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java [hunk 1]
```diff
@@ -32,12 +32,12 @@
...
     @Override
-    public void report( long transactions, long timeSlotMillis, long timeElapsedMillis )
+    public void report( long transactions, long elapsedTime)
     {
-        reports.add( ((double) transactions / (double) timeSlotMillis) );
+        reports.add( ((double) transactions / (double) elapsedTime) );
     }
 
-    public void assertUniformThroughput( PrintStream out )
+    public void assertThroughput( PrintStream out )
```
Comment (davidegrohmann) [inline stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java:40]: Don't we still assert that the throughout is somehow uniform?

File: stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java [hunk 2]
```diff
@@ -60,19 +60,23 @@ public void assertUniformThroughput( PrintStream out )
...
             {
                 System.err.println( "Found a time slot when throughput differs too much wrt the average:" );
                 System.err.println( "\tAverage="+average+", current="+report+", tolerance="+tolerance );
-                success = false;
+                nonMatchingCounter++;
             }
         }
-        assertTrue( success );
+        int nonSuccessPercentage = (int) (( nonMatchingCounter / (double) reports.size() ) * 100);
+        System.out.println("Non successful percentage is: " + nonSuccessPercentage);
```
Comment (davidegrohmann) [inline stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java]: code format

File: stresstests/src/test/java/org/neo4j/kernel/stresstests/workload/Workload.java [hunk 3]
```diff
@@ -88,15 +92,21 @@ public void run( long runningTimeMillis, TransactionThroughput throughput )
...
     public void close()
     {
-        executor.shutdown();
+        try
+        {
+            executor.shutdown();
+            executor.awaitTermination( 10, TimeUnit.SECONDS );
+        } catch ( Exception e ) {
+            throw new RuntimeException( e );
+        }
```
Comment (davidegrohmann) [inline stresstests/src/test/java/org/neo4j/kernel/stresstests/workload/Workload.java]: formatting is wrong

**Comment by davidegrohmann** [comment]
Small cosmetics issues, this looks good to me otherwise.

File: stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java (hunk 1 above)
Comment (MishaDemianenko) [inline stresstests/src/test/java/org/neo4j/kernel/stresstests/TransactionThroughputChecker.java:40]: I not really sure that we have normal distribution over there. That we can clearly see in the beginning when throughput is lower then all the rest. Plus final assumption to have 80% of measurements in a average +- range sounds more like normal distribution. Thats why word `uniform` was removed.

**Pull request merged**